src/azure-cli/azure/cli/command_modules/appconfig/tests/latest/export.json

```diff
@@ -1,6 +1,20 @@
 {
-  "FeatureManagement": {
-    "Beta": false
+  "BackgroundColor": "red",
+  "Language": "spanish",
+  "Langugage": "English",
+  "Settings:BackgroundColor": {
+    "cars": {
+      "Toyota": "Acura"
+    },
+    "ship": [
+      "ship1",
+      "ship2",
+      "ship3"
+    ]
```

Comment:
Why do you need to modify this file?

Reply:
don't know, life run modify it automatically.